command/init_test.go

```diff
@@ -374,6 +374,24 @@ func TestInit_backendConfigFile(t *testing.T) {
 			t.Fatalf("wrong error: %s", ui.ErrorWriter)
 		}
 	})
+
+	// the backend config file must be a set of key-value pairs and not a full backend {} block
```

Comment:
Please update this comment to reflect the test behavior.

Reply:
Whoops, copy paste. Good catch, i'll update